docs/photoniq/prerendering/best-practices-prerendering.md

```diff
@@ -5,26 +5,22 @@ title: Prerendering Best Practices
 
 Prerendering is an optimization technique that generates static HTML pages from dynamic content to improve website performance. The following best practices help you get the most out of Prerendering.
 
-## Selective Prerendering
-
-To maximize the benefits of prerendering, target specific areas of your website that will gain the most from this optimization technique. Typically, these are high-traffic pages or sections featuring dynamic content that seldom changes. By focusing on these areas, you ensure efficient use of resources and faster load times for your users.
-
 ## Domain Configuration
 
 Be very careful when configuring the domains for prerendering. Incorrect settings can result in caching problems and the delivery of outdated content. Make sure the specified domains are accurate and correspond to the sections of your site that you intend to prerender.
 
 ## Use the API for Caching and Recaching
 
-Take advantage of the prerendering service's API to manually cache or recache pages. This ensures that crawlers always access up-to-date content, and allows you greater control over your caching strategy.
+Take advantage of the prerendering service's API to programatically cache or recache pages. This ensures that crawlers always access up-to-date content, and allows you greater control over your caching strategy. You can also manually purge the cache in the web console.
```

Comment:
As I mentioned above. I think it is better to mention how caching can improve the pretender experience. Something like:
`Take advantage of the caching feature to serve the static version of your website's page faster. By using this feature, the pages are cached and served to the bots without the need of prerendering them again. Configure the cache expiration time according to how frequently your website is modified.`